kopf/_cogs/structs/patches.py

```diff
@@ -80,17 +81,38 @@ def spec(self) -> SpecPatch:
     def status(self) -> StatusPatch:
         return self._status
 
+    @property
+    def original(self):
+        return self._original
+
+    @original.setter
+    def original(self, value):
+        self._original = value
+
```

Comment:
I am worried that an extra public signature is exposed here. Can we avoid this? For example, by instantiating the patch with an optional kwarg (positional args are taken by the dict-like signatures):

```python
patch = Patch()
patch = Patch(body=raw_body)
patch = Patch({...})
patch = Patch({...}, body=raw_body)
```

This will let the patch keeping the original body for reference, but not expose its existence to the users.

Reply:
Yeah; sounds good. I have updated the PR with changes to use kwargs here, and removed the `@setter` method.

Reply:
And what is the supposed used for the getter? (`body` kwarg is provided to handlers together with other body parts.)

My point was that deprecating and removing the public interfaces is much more difficult than just not adding them in the first place — if possible (i.e when there are better ways).

Besides the `.original`, LGTM.

Reply:
Sorry, I guess am not completely sure what you are asking. Are you saying to get rid of the `original` property?

Reply:
Yes, it would be better to remove it — unless there is a reason why it is exposed to users. (And to keep the protected `._original` field as it is.)

Reply:
Got it; removed.